src/docs/platform-architecture-reference/platform-architecture-diagram.md

```diff
@@ -18,25 +18,13 @@ content_owner: Olena Mitovska
 sort_order: 2
 
 ---
-<!-- omit in toc -->
-# Platform Architecture Diagram
-Last updated: **August 22, 2023**
+# Platform architecture diagram
+Last updated: **May 8, 2024**
 
-The platform architecture diagram provides a clear visual representation of the layout within the datacenter. It showcases the locations of different clusters and outlines the diverse range of services and features that each cluster offers.  
+The platform architecture diagram provides a clear visual representation of the layout within the datacentre. It showcases the locations of different clusters and outlines the diverse range of services and features that each cluster offers.  
 
-To view the diagram you will need to follow the directions below as it is content protected by IDIR login. The diagram serves as a valuable tool for understanding how the datacenter's resources are organized and how various services are distributed across the infrastructure:
+[View the diagram](https://digital.gov.bc.ca/cloud/services/private/internal-resources/diagram/) (protected by IDIR login). The diagram serves as a valuable tool for understanding how the datacentre's resources are organized and how various services are distributed across the infrastructure.
 
-1. Head on over to the [Private cloud](https://digital.gov.bc.ca/cloud/private/) main page
-
-2. Find the section: **Internal resources**. Please note that this section is password-restricted for private cloud resources (IDIR protected)
-
-3. Click on the button  **Login with Keycloak** 
-
-4. Log in with IDIR
-
-5. Find the section: **Platform architecture diagram**
-
```

Comment:
What is the intent of removing these instructions?

Reply:
The 'direct link' Connor shared in the other PR still requires IDIR login for me (at least for me from outside Gov network). Maybe Connor doesn't realise this or was already logged in? I think it's worth explaining the steps and they should stay in there.

Reply:
My mistake here, after testing this AM it looks like the direct link does now redirect to the Platform Architecture diagram. So no need for the extra instructions.

Reply:
I tried the [https://digital.gov.bc.ca/cloud/services/private/internal-resources/diagram/](https://digital.gov.bc.ca/cloud/services/private/internal-resources/diagram/) link. I was prompted to login with IDIR but then the diagram displayed for me.

I think the new instructions describe the process appropriately.

Reply:
I uploaded a video on the other PR that I'll attach here. When using firefox, the site seems to direct users to the top level digital.gov.bc.ca page rather than redirecting. I wonder if we should include one more step to encourage people to try the link again after logging in?

https://github.com/bcgov/platform-developer-docs/assets/99685508/a06d9e8e-d217-463b-93da-63acc6096fd8